scipy/signal/wigner.py

```diff
@@ -0,0 +1,99 @@
+"""Tools based on the Wigner quasiprobability distributions.
+"""
+
+import numpy as np
+from scipy.fft import fft
+
+
+__all__ = ['wvd']
+
+
+def wigner_ville(x, fs=1.0, resolution=1, window_size=None):
```

Comment:
Would you be willing to add annotations for the new function?
It's signature seems to be rather straightforward.
```suggestion
def wigner_ville(
    x: npt.ArrayLike, 
    fs: float = 1.0, 
    resolution: int = 1, 
    window_size: None | int = None,
) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
```
Note that this will also require the following imports:
``` python
from __future__ import annotations

from typing import Tuple, TYPE_CHECKING

if TYPE_CHECKING:
    import numpy.typing as npt
```

Reply:
I like that. I just had not seen type hints in any of the other scipy methods I used as a reference.